go/tools/builders/link.go

```diff
@@ -50,6 +50,7 @@ func link(args []string) error {
 	flags.Var(&xdefs, "X", "A string variable to replace in the linked binary (repeated).")
 	flags.Var(&xstamps, "Xstamp", "Like -X but the values are looked up in the -stamp file.")
 	flags.Var(&stamps, "stamp", "The name of a file with stamping values.")
+	importPathConflictIsError := flags.Bool("importpath-conflict-is-error", false, "Whether importpath conflicts are errors.")
```

Comment:
Nit: use underscores instead of dashes to separate words in flag names.

Reply:
Done